packages/webapp/src/inductions/components/induction-step-endorsement.tsx

```diff
@@ -155,128 +171,156 @@ export const InductionStepEndorsement = (props: Props) => {
+                    <ActionButton
+                        disabled={isLoading || !isReviewed}
+                        onClick={submitEndorsement}
+                    >
+                        {isLoading ? "Submitting endorsement..." : "Submit"}
+                    </ActionButton>
+                </div>
             </div>
         </div>
-    ) : (
-        <div>
-            Waiting for all the endorsements to complete the induction process.
-        </div>
     );
 };
 
 interface DonationFormProps {
     isInvitee: boolean;
     isLoading: boolean;
+    setReviewStep: (step: "profile" | "video") => void;
     submitDonation: () => void;
 }
 const DonationForm = ({
     isInvitee,
     isLoading,
+    setReviewStep,
     submitDonation,
 }: DonationFormProps) => {
     return isInvitee ? (
         <div className="space-y-3">
             <Text>
-                This is your last chance to review your profile and make sure
-                everything is correct. If you want to proceed, click on the
-                below button to donate and complete your induction!
+                This is your last chance to review your profile for completeness
+                and accuracy. If anything needs to be corrected,{" "}
+                <Link onClick={() => setReviewStep("profile")}>click here</Link>
+                . Keep in mind that modifying your profile will require your
+                endorsers to submit their endorsements again.
             </Text>
-            <div className="w-max mx-auto">
-                <Button onClick={submitDonation} disabled={isLoading}>
+            <Text>
+                If everything looks good, click on the button below to make your
+                donation to the Eden community. Once completed, your membership
+                will be activated and your Eden NFTs will be minted and
+                distributed.
+            </Text>
+            <div className="pt-1">
+                <ActionButton disabled={isLoading} onClick={submitDonation}>
                     {isLoading
                         ? "Submitting donation..."
-                        : `I want to Donate ${assetToString(
-                              minimumDonationAmount
-                          )}`}
-                </Button>
+                        : `Donate ${assetToString(minimumDonationAmount)}`}
+                </ActionButton>
             </div>
         </div>
     ) : (
-        <div>
-            Please reach out to the invitee and let him know that his induction
-            is endorsed and it's only waiting for his donation to complete the
-            induction!
-        </div>
+        <Text>
+            This induction is fully endorsed! As soon as the prospective member
+            completes their donation to the Eden community, their membership
+            will be activated and their Eden NFTs will be minted and
+            distributed.
+        </Text>
     );
```

Comment:
great job on this! oh man, I'm really missing your screenshots, I just need to use my imagination to see how better it is now :(

Reply:
Thanks!